.bazelversion

```diff
@@ -1 +1 @@
-7.4.1rc2
+7.5.0rc3
```

Comment:
Changes: https://github.com/bazelbuild/bazel/compare/7.4.0...7.5.0rc3

- [ ] Wait until 7.5.0 stable is released before merging this.